nservicebus/testing/index_content_testing_[6,).partial.md

```diff
@@ -1,15 +1,37 @@
 ## Testing a handler
 
+Testing a message handler is done using the `TestableMessageHandlerContext` class provided by the `NServiceBus.Testing` package. This class implements `IMessageHandlerContext` and can be passed to the handler under test. After the handler is executed, the `TestableMessageHandlerContext` can be interrogated to assert that various actions (sending a message, publishing an event, etc.) occurred in the handler as expected.
+
+| Property | Description |
+|---------------------|----------------------------------------------------------------------------------|
+| `SentMessages` | A list of all messages sent by `context.Send()`. |
```

Comment:
Should this also mention that it includes all delayed and timeout messages as well as of now? This is supposed to change though (see https://github.com/Particular/NServiceBus.Testing/issues/127)

Reply:
ah, I see that you've already mentioned this in the saga section. Sounds like a good approach 👍